test/index.test.js

```diff
@@ -54,8 +54,8 @@ describe('Uploader Core', () => {
   it('Can parse environment variables', async () => {
     process.env.SOMETHING = 'red'
     process.env.ANOTHER = 'blue'
-    jest.spyOn(process, 'exit').mockImplementation(() => {})
     const log = jest.spyOn(console, 'log').mockImplementation(() => {})
+    const log = jest.spyOn(process, 'exit').mockImplementation(() => {})
```

Comment:
```suggestion
    jest.spyOn(process, 'exit').mockImplementation(() => {})
```
Do no try to code before caffeine 🤦